src/memento_test.jl

```diff
@@ -85,7 +109,7 @@ end
 """
     @test_throws(logger, extype, expr)
 
-Disables the `logger` and calls `@test_throws extype, expr`.
+Disables the `logger` and calls `@test_throws extype expr`.
```

Comment:
Can we reference the Base `@test_throws` macro here?

Reply:
I've some additional cross-referencing links in.

Reply:
Thanks.